packages/chopsticks/src/cli.ts

```diff
@@ -13,17 +14,34 @@ import { decodeKey } from './utils/decoder'
 import { dryRun } from './dry-run'
 import { dryRunPreimage } from './dry-run-preimage'
 import { isUrl } from './utils'
+import { logger } from './rpc/shared'
 import { runBlock } from './run-block'
 import { tryRuntime } from './try-runtime'
 
 dotenv.config()
 
+const CONFIGS_BASE_URL = 'https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/'
+
 const processConfig = async (path: string) => {
   let file: string
   if (isUrl(path)) {
     file = await axios.get(path).then((x) => x.data)
   } else {
-    file = readFileSync(path, 'utf8')
+    try {
+      accessSync(path, constants.R_OK)
```

Comment:
why is this needed?

Reply:
Make it easy to load some default configs. We can make configs to be more generic

Reply:
Do you mean accessSync or this PR?

Reply:
It's just checking if can access that path, not super important

Reply:
Yeah I was trying something else and left this there